.docker/autoline/docker-compose.stage.yml

```
@@ -0,0 +1,59 @@
+version: "3.8"
+
+services:
+  autoline_mariadb:
+    image: mariadb
+    container_name: autoline_mariadb
+    build:
+      context: ./
+      dockerfile: database/Dockerfile
+    restart: on-failure
+    ports:
+      - "3306:3306"
+    volumes:
+      - autoline_mariadb_data:/var/lib/mysql
+    env_file:
+      - ../../.env/db.env
+    networks:
+      - autoline_backend_network
+  autoline_backend:
```

A bit of space would be awesome
```suggestion
      - autoline_backend_network

  autoline_backend:
```